packages/server/database/types/Reflection.ts

```diff
@@ -1,8 +1,14 @@
+import {sql} from 'kysely'
 import extractTextFromDraftString from 'parabol-client/utils/draftjs/extractTextFromDraftString'
 import generateUID from '../../generateUID'
 import GoogleAnalyzedEntity from './GoogleAnalyzedEntity'
 import Reactji from './Reactji'
 
+export const toGoogleAnalyzedEntityPG = (entities: GoogleAnalyzedEntity[]) =>
+  sql<
+    string[]
+  >`(select coalesce(array_agg((name, salience, lemma)::"GoogleAnalyzedEntity"), '{}') from json_populate_recordset(null::"GoogleAnalyzedEntity", ${JSON.stringify(entities)}))`
```

Comment:
we have to be complex here to handle entities like `Google, Inc.` since that name is stored in a composite type & it has a comma in it.